packages/flutter_tools/lib/src/doctor.dart

```diff
@@ -4,6 +4,8 @@
 
 import 'dart:async';
 import 'dart:convert' show UTF8;
+import 'package:http/http.dart' as http;
+import 'package:xml/xml.dart' as xml;
```

Comment:
Consider moving these imports down in the "package imports" area.  (Below `archive`.)